onnxruntime/core/graph/graph_utils.h

```diff
@@ -16,6 +16,13 @@ namespace onnxruntime {
 
 namespace graph_utils {
 
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
+
+/** Returns the attribute of a Node with a given name. */
+const ONNX_NAMESPACE::AttributeProto* GetNodeAttribute(const Node& node, const std::string& attr_name);
+
```

Comment:
nit, can be moved below with AddInitializer to reduce number of #ifdef?

Reply:
this one is unfortunately used by the template GetRepeatedNodeAttributeValues(), which is above the other ifdef

Reply:
Can we move the other 2 functions up here in the same `#ifdef`?